internal/handlers/handler_test.go

```diff
@@ -115,6 +115,55 @@ func TestListDatabases(t *testing.T) {
 	}
 }
 
+func TestServerStatus(t *testing.T) {
+	t.Parallel()
+	ctx := testutil.Ctx(t)
+	pool := testutil.Pool(ctx, t)
+	l := zaptest.NewLogger(t)
+	shared := shared.NewHandler(pool, "127.0.0.1:12345")
+	sql := sql.NewStorage(pool, l.Sugar())
+	jsonb1 := jsonb1.NewStorage(pool, l)
+	handler := New(pool, l, shared, sql, jsonb1)
+
+	type testCase struct {
+		req  types.Document
+		resp types.Document
+	}
+
+	tc := &testCase{
```

Comment:
If you change this to using a map and iterating through the cases [see here](https://github.com/FerretDB/FerretDB/blob/b36e0a15a7c41121b02c599dfbb359fe0e83dae1/internal/handlers/handler_test.go#L49). This will make extending the tests easier in the future